app/org/sagebionetworks/bridge/dynamodb/DynamoInitializer.java

```diff
@@ -490,18 +488,6 @@ static void compareSecondaryIndices(String tableName, List<?> indices1, List<?>
                             " is changing the projection.");
                 }
 
-                if (!globalIndex1.getProvisionedThroughput().getReadCapacityUnits().equals(
-                        globalIndex2.getProvisionedThroughput().getReadCapacityUnits())) {
-                    throw new BridgeInitializationException("Table " + tableName + " global index " + indexName +
-                            " is changing the provisioned read capacity.");
-                }
```

Comment:
Just to be clear.  Throughput validate is being removed in this pull request.

But adding, deleting, renaming indices are operations with no small impact  and, in my opinion, should always go through code deploy and data migrations.